packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -41,12 +41,12 @@ export async function allowScreenCaptureAsync(): Promise<void> {
  * usePreventScreenCapture();
  * ```
  */
-export function usePreventScreenCapture(): void {
+export function usePreventScreenCapture(tag = 'default'): void {
   useEffect(() => {
-    preventScreenCaptureAsync();
+    preventScreenCaptureAsync(tag);
 
     return () => {
-      allowScreenCaptureAsync();
+      allowScreenCaptureAsync(tag);
     };
   }, []);
```

Comment:
Need to re-run the hook when the tag changes. (Add a regression test for this!)

Reply:
All these changes were addressed *except* for the hook regression test. Still need to figure out how to accurately mock the hooks

Reply:
Alright, these are added now and will merge once tests pass (besides client_android which is failing on master). Thanks @ide for all the reviews & feedback!